code/modules/paperwork/desk_bell.dm

```diff
@@ -106,3 +106,19 @@
 /obj/structure/desk_bell/speed_demon
 	desc = "The cornerstone of any customer service job. This one's been modified for hyper-performance."
 	ring_cooldown_length = 0
+
+/obj/structure/desk_bell/MouseDrop(obj/over_object, src_location, over_location)
+	if((!istype(over_object, /obj/vehicle/ridden/wheelchair)))
+		return
+	if(!Adjacent(over_object) || !Adjacent(usr))
+		return
+	var/obj/vehicle/ridden/wheelchair/target = over_object
+	if(target.bell_attached)
+		usr.balloon_alert(usr, "already has a bell!")
+		return
+	usr.balloon_alert(usr, "attaching bell...")
+	if(!do_after(usr, 5))
+		return
+	target.attach_bell(src)
+	qdel(src)
```

Comment:
This doesn't make sense. We're attaching the src bell to the target, which uses and stores the object directly, then we're *qdeleting*? This is guaranteed to hard-del and be weird. `attach_bell` should just put `src` in the contents instead.